public/app/plugins/datasource/prometheus/components/monaco-query-field/MonacoQueryField.tsx

```diff
@@ -199,6 +202,10 @@ const MonacoQueryField = (props: Props) => {
           editor.addCommand(monaco.KeyMod.Shift | monaco.KeyCode.Enter, () => {
             onRunQueryRef.current(editor.getValue());
           });
+
+          editor.addCommand(monaco.KeyMod.CtrlCmd | monaco.KeyCode.KeyK, function () {
```

Comment:
@Elfo404 🤷 😭

Reply:
Test with feature flag off

Reply:
Test with feature flag off

Reply:
check to see if our keybinding service sees the K part
emit the fake event and see if we can get existing toggles to pick it up

Reply:
check to see if our keybinding service sees the K part
emit the fake event and see if we can get existing toggles to pick it up